tests/xlsx/xlsx_export.test.ts

```diff
@@ -75,6 +75,7 @@ const simpleData = {
         A38: { content: `='<Sheet2>'!B2` },
         A39: { content: `=A39` },
         A40: { content: `=(1+2)/3` },
+        A41: { content: `Indented text`, style: 12 },
```

Comment:
The commit says that the xlsx import works. I don't think I saw it implemented ? Or at least it's not tested